test/test_cuda.py

```diff
@@ -401,6 +401,8 @@ def tmp(t):
     ('rsqrt', lambda t: constant_tensor_add(1, small_3d(t)), lambda t: [], None, float_types),
     ('sinh', lambda t: tensor_clamp(small_3d(t), -1, 1), lambda t: [], None, float_types),
     ('tan', lambda t: tensor_clamp(small_3d(t), -1, 1), lambda t: [], None, float_types),
+    ('__lshift__', lambda t: torch.pow(2, torch.arange(1., 5.)), lambda t: [2], None, [torch.FloatTensor]),
+    ('__rshift__', lambda t: torch.pow(2, torch.arange(4., 8.)), lambda t: [4], None, [torch.FloatTensor]),
```

Comment:
Are these sufficient @zou3519 ?

Reply:
I have added a tests for multiple types. @zou3519

Reply:
Yes these tests should be sufficient!